packages/syntax-core/src/Typography/Typography.tsx

```diff
@@ -30,6 +30,7 @@ const Typography = ({
   children,
   color = "gray900",
   inline = false,
+  lineClamp = undefined,
```

Comment:
@jliotta this will be the default already, so no need to set it again.

So just `lineClamp`